src/config.ts

```diff
@@ -101,7 +101,7 @@ export interface KafkaConfigurations {
 
 const buildAppContext = (): AppConfig => {
   dotenv.config();
-  logger.info('building app context');
+  console.info('building app context');
```

Comment:
curious why use console if there's a logger present?

Reply:
Config is created before logger...i found this out the slow way 🙂